governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022
+summary: Coordinate and execute MakerDAO branded events at Crypto/DEV Conference locations.
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp32-adding-events-core-unit-events-001/13780
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-11T16:00:00
+---
+# Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Adding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP32
```

Comment:
"MIP39c2-SP32: Adding Events Core Unit (EVENTS-001)".

We should do the same with the rest, too